tools/install.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+SCRIPT_DIR="$( cd "$(dirname "$0")" ; pwd -P )"
+
+# installation of the ProActive distribution
+ 
+read -e -p "Directory where to install the scheduler: " -i "/opt/proactive" PA_ROOT
+
+mkdir -p $PA_ROOT
+rm -f $PA_ROOT/default
+
+CURRENT_PADIR="$(dirname "$SCRIPT_DIR")"
+PA_FOLDER_NAME="$(basename "$CURRENT_PADIR")"
+
+/usr/bin/cp -f -R $CURRENT_PADIR $PA_ROOT
+
+ln -s -f $PA_ROOT/$PA_FOLDER_NAME $PA_ROOT/default
+
+# creation of the proactive user
+
+read -e -p "Name of the user starting the ProActive service: " -i "proactive" USER
+
+echo "Do you want to create this user ?"
+select yn in "Yes" "No"; do
+    case $yn in
+        Yes ) 
+		useradd -m $USER;
```

Comment:
If a user doesn't know where installation directory is but knows under which user the process is executing (e.g. `ps aux`) he can easily identify the installation directory of proactive service.